tests/model_hub_tests/tensorflow/requirements.txt

```diff
@@ -5,3 +5,5 @@ pytest-html
 tensorflow
 tensorflow-hub
 tensorflow-text
+transformers
+tf-sentence-transformers
```

Comment:
why do we need this?

Reply:
These are libraries that have TFAutoModel and SentenceTransformer classes, which are needed to load Hugging Face models.

transformers we use in pytorch as well: https://github.com/openvinotoolkit/openvino/blob/ac04160586132b9188237b465674b324f997272d/tests/model_hub_tests/torch_tests/requirements.txt#L24